src/sentry/incidents/utils/metric_issue_poc.py

```diff
@@ -43,16 +44,21 @@ def to_dict(self) -> dict[str, Any]:
     "count_unique(tags[sentry:user])": "Number of users affected",
     "percentage(sessions_crashed, sessions)": "Crash free session rate",
     "percentage(users_crashed, users)": "Crash free user rate",
+    "failure_rate()": "Failure rate",
+    "apdex()": "Apdex score",
 }
 
 
-def construct_title(alert_rule: AlertRule) -> str:
+def construct_title(alert_rule: AlertRule, status: int) -> str:
```

Comment:
i think this function is starting to get a bit confusing. could we instead use composition and construct the final string based off of each component?